AirCasting/SessionsSynchronization/Controller/SessionSynchronizationViewModel.swift

```diff
@@ -0,0 +1,25 @@
+// Created by Lunar on 09/09/2021.
+//
+
+import Foundation
+
+protocol DefaultSessionSynchronizer {
```

Comment:
it should have a "ViewModel" in the name. I suggest protocol named `SessionSynchronizationViewModel` and implementation with `Default` prefix